### DISCUSSION TITLE
3.0.2 version compatibility

### DISCUSSION THREAD

**Discussion by balnaborju** [esp8266/Arduino#8701] [General]
Hi,

Does the current 3.0.2 version work with the new Arduino 2.0.0 IDE ?

Thanks

**Comment by mhightower83** [comment]
@balnaborju, From my quick look, java based plugin tools like EspExceptionDecoder and others are not supported on Arduino IDE 2.0.
This post looks relevant: https://forum.arduino.cc/t/tool-to-upload-data-to-spiffs-is-missing/699986
> Edited: this one is even more relevant: https://github.com/arduino/arduino-ide/issues/58#issuecomment-1248162975
It appears at the present there is no plugin API support.

Otherwise from my limited use, I can do builds and ".bin" downloads with IDE 2.0.0. (I had problems with IDE 2.0.1, sluggish in opening files and screens not coming up, etc.)
